numpy/f2py/crackfortran.py

```diff
@@ -1127,7 +1127,6 @@ def analyzeline(m, case, line):
                         (m.group('this'), ll[:i]))
                 ll = ll + ','.join(groupcache[groupcounter]['args'])
         if i < 0:
-            i = 0
```

Comment:
This is valid removal.

Reply:
hey, I commited your reviewed changes. Can you tell if you are satisfied ? 
The azure lint test seems to fail, reason being `outmess(f'get_parameters[TODO]: implement evaluation of complex expression {v}')`, Can you check?

Reply:
That's a "line too long" error. You'll need to split the line, for instance:
```
outmess('get_parameters[TODO]:'
        f' implement evaluation of complex expression {v}')
```
(untested).

Reply:
The removal looks to be gone, wasn't it valid?